asserts/confdb.go

```diff
@@ -151,6 +153,21 @@ func (cc *ConfdbControl) Prerequisites() []*Ref {
 	}
 }
 
+// NewConfdbControl returns an empty confdb-control assertion.
+func NewConfdbControl(brand, model, serial string) *ConfdbControl {
```

Comment:
is never acceptable to have a way to construct unsigned assertions, I'm not sure if this meant for test or what but we shouldn't have it here in this form, tests can use FakeAssertion

Reply:
For this one, I thought it would be useful to have a method for us to create a new unsigned assertion when the API is first called since we won't have any `confdb-control` in the assertions database. Something like [this](https://github.com/st3v3nmw/snapd/blob/11b14dd1d31673232043fcbfcfaf933ca10162b0/daemon/api_confdb.go#L216) which would then [sign and acknowledge](https://github.com/st3v3nmw/snapd/blob/11b14dd1d31673232043fcbfcfaf933ca10162b0/daemon/api_confdb.go#L204C12-L204C40)

But there might be a better way to do it

Reply:
this is mostly ok because that assertion gives not permissions but then we need a very different name for this?

Reply:
but seems it is used to then mutate the assertion which is a no-no in itself